### PR TITLE
az-disk: add page

### PR DESCRIPTION
**Pull request by brokoli777**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

#6141

### CODE REVIEW

File: pages/common/az-disk.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+`az disk list --resource-group {{resource_group}}`
+
+- Delete a managed disk:
+
+`az disk delete --resource-group {{resource_group}} --name {{disk_name}}`
+
+- Grant access to a managed disk (for export):
+
+`az disk grant-access --resource-group {{resource_group}} --name {{disk_name}} --access-level Read --duration-in-seconds {{seconds}}`
```
Comment (pixelcmtd) [inline pages/common/az-disk.md:21]: Does `--access-level` also accept other values? (if it does, maybe it should be `{{Read}}`?)

Comment (spageektti) [reply pages/common/az-disk.md:21]:
```suggestion
`az disk grant-access --resource-group {{resource_group}} --name {{disk_name}} --access-level {{Read|Write}} --duration-in-seconds {{seconds}}`
```
It does -> https://learn.microsoft.com/pl-pl/cli/azure/disk?view=azure-cli-latest#az-disk-grant-access

File: pages/common/az-disk.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- List managed disks in a resource group:
+
+`az disk list --resource-group {{resource_group}}`
+
+- Delete a managed disk:
+
+`az disk delete --resource-group {{resource_group}} --name {{disk_name}}`
+
+- Grant access to a managed disk (for export):
```
Comment (spageektti) [inline pages/common/az-disk.md:19]:
```suggestion
- Grant read or write access to a managed disk (for export):
```

**Comment by brokoli777** [comment]
Thank you for the suggestions! I have committed them.

**Review by spageektti — APPROVED** [review]

**Pull request merged**